managed/services/management/dbaas/kubernetes_server.go

```diff
@@ -446,19 +467,22 @@ func (k kubernetesServer) installDefaultOperators(operatorsToInstall map[string]
 			channel = "stable-v1"
 		}
 		operatorName := "percona-xtradb-cluster-operator"
-		params := kubernetes.InstallOperatorRequest{
-			Namespace:              namespace,
-			Name:                   operatorName,
-			OperatorGroup:          operatorGroup,
-			CatalogSource:          catalogSource,
-			CatalogSourceNamespace: catalogSourceNamespace,
-			Channel:                channel,
-			InstallPlanApproval:    v1alpha1.ApprovalManual,
-		}
+		_, err := kubeClient.GetSubscription(ctx, namespace, operatorName)
```

Comment:
It would be great to wait until VM's CRDs will be present in a cluster. There can be a corner case when we start provisioning monitoring and applying rbac rules or starting kube-state-metrics will fail because CSV is in Ready to install or installing state.